news/2024/2024-11-17-kel-lan-tournament-3.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: KEL LAN Tournament 3
```

Comment:
feels barren with just the tournament name, and signups didnt open that long ago so its fine to use this as an announcement 
```suggestion
title: "KEL LAN Tournament 3: Signups Now Open!"
```

Reply:
Signups are opened since October 8th, and I don't think adding "Signups Now Open!" is a good idea tbh

Reply:
maybe something about signups closing soon then? @Walavouchey thoughts on this?

Reply:
well signups are closing on December 2nd so this might work

Reply:
eh, pretty sure we've had similar tournament titles before (without subtitle)

i think this would work too

```suggestion
title: "KEL LAN Tournament 3: Registrations Open!"
```

Reply:
the above suggestions is fine as well